openslides_backend/action/default_schema.py

```diff
@@ -13,20 +13,22 @@ class DefaultSchema:
     def __init__(self, model: Model) -> None:
         self.model = model
 
-    def get_create_schema(
-        self, properties: Iterable[str], required_properties: Iterable[str]
+    def get_default_schema(
+        self,
+        required_properties: Iterable[str] = [],
+        optional_properties: Iterable[str] = [],
     ) -> Dict:
         """
         Returns a default create schema with properties and required properties as given.
         """
         return {
             "$schema": schema_version,
-            "title": f"New {self.model} schema",
-            "description": f"An array of new {self.model} objects.",
             "type": "array",
             "items": {
                 "type": "object",
-                "properties": self.model.get_properties(*properties),
+                "properties": self.model.get_properties(
+                    *required_properties, *optional_properties
+                ),
                 "required": list(required_properties),
                 "additionalProperties": False,
             },
```

Comment:
Please add `"minItems": 1` and `"uniqueItems": True` here.

Reply:
It's already there, seems like github clipped it ;)